ios/browser/brave_web_client.mm

```diff
@@ -20,6 +23,15 @@
 BraveWebClient::~BraveWebClient() {
 }
 
+void BraveWebClient::AddAdditionalSchemes(Schemes* schemes) const {
+  schemes->standard_schemes.push_back(kChromeUIScheme);
+  schemes->secure_schemes.push_back(kChromeUIScheme);
+}
+
+bool BraveWebClient::IsAppSpecificURL(const GURL& url) const {
+  return url.SchemeIs(kChromeUIScheme);
```

Comment:
Should we handle `brave://` as well? In the future?

Reply:
No idea how we can add `brave` scheme. So far it seems to load only with `chrome` scheme even if I added `brave` there. I would probably have to override a bunch of stuff to get it to translate `chrome` to `brave`.